spaces/importers/units.py

```diff
@@ -115,6 +115,10 @@ def _update_counters(self, created: bool):
 
         self.update_counter += 1
 
+    def _reset_counters(self):
```

Comment:
This function might be useful, but is not currently used anywhere. I would not add any unnecessary code that is not used anywhere just because it might be useful.

Reply:
Very true, I'll get rid of this